Exiled.Events/EventArgs/Player/TransmittingEventArgs.cs

```diff
@@ -48,11 +49,11 @@ public TransmittingEventArgs(Player player, VoiceModuleBase voiceModule)
         /// <summary>
         ///     Gets a value indicating whether or not the player is transmitting.
         /// </summary>
-        public bool IsTransmitting { get; }
+        public bool IsTransmitting => true;
```

Comment:
I think obsolete was right here. Idk why you removed it

Reply:
Idk i was probably tired :trollface: